.github/DISCUSSION_TEMPLATE/gsoc-2025-proposal-reviews.yml

```diff
@@ -1,4 +1,4 @@
-title: "GSoC’23 Proposal Review | {{Project Name}} ({{Your Name}})"
+title: "GSoC’25 Proposal Review | {{Project Name}} ({{Your Name}})"
 body:
 - type: input
   attributes:
```

Comment:
Below: "Google doc" --> "Google Doc" (both in label and description)

Also, in the "review section" category let's add a third option: "Both 3.1: WHAT and 3.2: HOW"

Reply:
Done